webcompat/static/js/lib/issues.js

```diff
@@ -484,6 +485,10 @@ issues.MainView = Backbone.View.extend({
   },
   render: function() {
     this.$el.fadeIn();
+  },
+
+  handleKeyShortcuts: function(e) {
+     Mousetrap.bind('mod+enter', this.addNewComment() ,'keydown');
```

Comment:
(this has the advantage of making an obvious place for any new complex keyboard shortcuts to go)